pxr/usd/usdMtlx/api.h

```diff
@@ -1,11 +1,28 @@
 //
-// Copyright 2018 Pixar
+// Copyright 2017 Pixar
 //
-// Licensed under the terms set forth in the LICENSE.txt file available at
-// https://openusd.org/license.
+// Licensed under the Apache License, Version 2.0 (the "Apache License")
+// with the following modification; you may not use this file except in
+// compliance with the Apache License and the following modification to it:
+// Section 6. Trademarks. is deleted and replaced with:
 //
-#ifndef PXR_USD_USDMTLX_API_H
-#define PXR_USD_USDMTLX_API_H
+// 6. Trademarks. This License does not grant permission to use the trade
+//    names, trademarks, service marks, or product names of the Licensor
+//    and its affiliates, except as required to comply with Section 4(c) of
+//    the License and to reproduce the content of the NOTICE file.
+//
+// You may obtain a copy of the Apache License at
```

Comment:
Note that we recently updated our license text, so you'll want to go with the top-of-tree license text.  If you're running usdSchemaGen with an old USD version (< 24.08) that might put the wrong text in.